src/Findologic/Response/Xml21/Filter/Values/FilterValue.php

```diff
@@ -18,11 +21,14 @@ class FilterValue extends Struct
     /** @var TranslatedName */
     private $translated;
 
-    public function __construct(string $id, string $name)
+    public function __construct(string $id, string $name, ?string $filterName = null)
```

Comment:
When would a filter name be null? This looks like a case that should never happen, as we always should use the internal filter name.

Reply:
This is null because we are not setting the ID for all filter values. Just the ones where we need it to be. For e.g the category filter does not work with IDs, as we are already having a parent category name in the value with an `_` to make it unique. The uuid is generated only for the values in which we need a unique ID for it.

Reply:
I see. Please add a docblock why `$filterName` is necessary. This way future developers understand why you sometimes need the name of the filter and sometimes not.